Dockerfile

```diff
@@ -21,6 +21,9 @@ ENV STF_PROVIDER_HOST=
 ENV STF_PROVIDER_CONNECT_SUB=
 ENV STF_PROVIDER_CONNECT_PUSH=
 
+ENV STF_PROVIDER_PORTS_WAIT=300
```

Comment:
it is prohibited to name env vars using `STF_PROVIDER_` prefix if they are not part of stf cli...
stf couldn't start erroring that ports_wait is not defined.
Let's name 
```
STF_SERVICES_WAIT_TIMEOUT=300
STF_SERVICES_WAIT_RETRY=30
```